ledger/src/blockstore_db.rs

```diff
@@ -196,13 +194,10 @@ impl Rocks {
             new_cf_descriptor::<columns::MerkleRootMeta>(options, oldest_slot),
         ];
 
-        // If the access type is Secondary, we don't need to open all of the
+        // If the access type is read-only, we don't need to open all of the
```

Comment:
nit: Not to punish a good deed but since we're cleaning up, this logic might be better off in `AccessType`. Ie, maybe a function like `AccessType::must_open_all_column_families() -> bool`

Reply:
Yeah fair enough, I like what you did so marking this one resolved